packages/daylight/start.go

```diff
@@ -121,11 +120,28 @@ func savePid() error {
 		log.WithFields(log.Fields{"pid": pid, "error": err, "type": consts.JSONMarshallError}).Error("marshalling pid to json")
 		return err
 	}
-	return ioutil.WriteFile(conf.GetPidFile(), PidAndVer, 0644)
+
+	return ioutil.WriteFile(conf.Config.GetPidPath(), PidAndVer, 0644)
+}
+
+func CreateLockFile() error {
+	return ioutil.WriteFile(conf.Config.LockFilePath, []byte{}, 0644)
 }
 
 func delPidFile() {
-	os.Remove(conf.GetPidFile())
+	os.Remove(conf.Config.GetPidPath())
+}
+
+func DelLockFile() error {
+	return os.Remove(conf.Config.LockFilePath)
+}
+
+func IsLockFileExists() bool {
```

Comment:
this is not correct at all, unix-style file locking should use fcntl.Flock syscall.
For example see https://github.com/tgulacsi/go-locking/blob/master/locking.go